index.js

```diff
@@ -38,11 +38,11 @@ class Tts extends NativeEventEmitter {
     return TextToSpeech.voices();
   }
 
-  speak(utterance, voiceId) {
+  speak(utterance, params, voiceId) {
```

Comment:
I think `params` should be added as a third parameter for the backward compatability? Any reason for this to be aded as the second parameter?

Reply:
If `params` is at second parameter, wouldn't `speak(utterance, {
  KEY_PARAM_STREAM: STREAM_VOICE_CALL
})` 
cause problem?

Reply:
Hum, I agree that it seems little ugly. However, I think we'd better apply the third parameter here and use `speak(utterance, undefined, { KEY_PARAM_STREAM: STREAM_VOICE_CALL });` not to diverge from the upstream repository.

@ghsdh3409 Any opinion on this?

Reply:
Yes, I agree with @ohtangza. For compatibility, `params` should be located at third argument.